portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase

```diff
@@ -113,6 +113,82 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: Legacy | Automation ID: LRAC-10257 | Test Summary: Assert total amount of views in all cards"
```

Comment:
Hey @calvinctvo, In the pattern that Alex define for the description of the tests, in the first part, put where the tests came from, if it was a story or a bug or if we don't have this definition and put Legacy. In your case it originated from a bug so I think you could put it like this:

`@description = "Bug: LRAC-10257| Automation ID: LRAC-10257 | Test Summary: Assert total amount of views in all cards"`

What do you think?